app/assets/javascripts/views/export_csv_view.js

```diff
@@ -2,30 +2,34 @@
 ELMO.Views.ExportCsvView = class ExportCsvView extends ELMO.Views.ApplicationView {
   get events() {
     return {
-      'click #response_csv_export_options_download_media': 'calculateMediaSize'
+      'click #response_csv_export_options_download_media': 'calculateDownloadSize',
+      'click #response_csv_export_options_download_xml': 'calculateDownloadSize'
     };
   }
 
   initialize() {
     $(".calculating-info").hide();
     $(".error-info").hide();
     $(".media-info").hide();
+    $("input#response_csv_export_options_download_csv")[0].checked = true;
   }
 
-  async calculateMediaSize(event) {
+  async calculateDownloadSize(event) {
+    let downloadType = "";
     if (($(event.target)).is(':checked')) {
       $("input[type=submit]").prop("disabled", true);
-      await this.spaceLeft();
-      $(".media-info").show();
+      downloadType = $(event.target)[0].name.includes("xml") ? "xml" : "media";
+      await this.spaceLeft(downloadType);
+      downloadType == "media" ? $(".media-info").show() : "";
     } else {
       this.enableSubmitButton();
       $(".media-info").hide();
       $(".error-info").hide();
     }
   }
 
-  async spaceLeft() {
-    $(".calculating-info").show();
+  async spaceLeft(downloadType) {
+    downloadType == "media" ? $(".calculating-info").show() : "";
```

Comment:
Interesting bug: if you check media FIRST then also check and/or uncheck xml, then the "total size" text gets modified/hidden. I think this JS file may need a bit more refactoring to play nicely together.

Suggestion: instead of making media a special case, you could add an additional "total size" row for xml also; the two different rows can be shown/hidden separately based on the checkboxes. Let me know if this is clear or I can explain verbally 😅

Reply:
fixed! makes sense!